### DISCUSSION TITLE
Success version of the Alert component

### DISCUSSION THREAD

**Discussion by Fazle-Jahangir** [twilio-labs/paste#2596] [General] [answered]
**Background**: Communicating the end of life for minor versions to users, removing them from the versioning page. We are also implementing modals for various scenarios notifying users when a version is nearing end of life. 

**Question**: I would like to find out if a **Success** of the **Alert** component has ever been considered? There is a valid use case here in to use a success alert instead of a neutral alert because the user cannot act upon this information. They have already successfully updated to the latest version. We are sticking with the neutral alert for now but I'm curious to hear your thoughts on this. 

<img width="957" alt="Screenshot 2022-08-08 at 18 00 04" src="https://user-images.githubusercontent.com/100706574/183472931-ed9c08f3-9c6f-4d56-bc1c-b462e4b04c9b.png">

**Comment by loreina** [comment, marked as answer]
Hi @Fazle-Jahangir, we don't have plans to add a success variant to Alert right now. We'll be adding a Callout component to Figma that has a success variant that we'd recommend for this use case. I'll follow up once that's added to the library.
